### DISCUSSION TITLE
encore delete app?

### DISCUSSION THREAD

**Discussion by rscorer** [encoredev/encore#1060] [Help] [answered]
I have managed to create a few copies of the demo apps, and wanted to clean up my "workspace" but cannot find a way to delete them...

Is there an `encore delete app` command?

Thanks

**Comment by marcuskohlberg** [comment, marked as answer]
Welcome!
Right now you delete apps via the Cloud Dashboard. Do this by opening your app on [app.encore.dev ](https://app.encore.dev)and click on **Settings**. Then in **General settings** you'll find the section **Delete this app**

The url will be:
https://app.encore.dev/your-app-id/settings/app

Hope that helps!

**Reply by rscorer** [reply]
Thanks, that worked great. Would be nice to have a direct link to the Cloud Dashboard, I found I had to choose Upload to Cloud then choose General from that dashboard. Maybe I missed the direct cloud dashboard link? `¯\\_(ツ)_/¯ `

**Reply by marcuskohlberg** [reply]
Sorry you had some trouble. Not sure what you mean exactly though. How do you mean "Upload to Cloud"?

If you were using the local development dashboard, there is a link to "Cloud Dashboard" in the top navigation from there.

**Reply by rscorer** [reply]
You are right -- I had forgotten about the local development dashboard -- so many dashboards!!

I had meant "Connect to Cloud" 

Looking forward to spend more time getting myself used to encore, it does seem it will be a useful ecosystem for me.

**Reply by marcuskohlberg** [reply]
I don't think you need to connect to cloud in order to delete an application, so if there's something confusing in the UX we should address that. Do you recall where you were reading to get this impression?

Also curious, if you were not using the local dashboard, what "workspace" did you refer to when you mentioned wanting to clean up?

**Reply by rscorer** [reply]
I was hoping for a command line instruction to delete the app, but using the local dashboard works fine - my apologies for being confusing, I was trying to clean up my sample projects without really concentrating. Entirely my bad.

Now, I just go to my initials in the local dashboard and choose Settings then choose General. Hit Delete... Easy!

There's even a convenient list of all my apps, honestly where was my head?!

**Reply by marcuskohlberg** [reply]
Thanks, having a CLI command would also be useful.
